tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/CalendarEventReplyMailProcessor.scala

```diff
@@ -63,18 +63,18 @@ import reactor.core.scheduler.Schedulers
 import scala.jdk.CollectionConverters.CollectionHasAsScala
 import scala.util.{Failure, Success, Try}
 
-object CalendarEventReplyPerformer {
+object CalendarEventReplyMailProcessor {
```

Comment:
The `CalendarEventReplyPerformer` class was created before we have two modes (standalone + CalDAV). It handles reply emails, queuing, and JMAP responses. 
However, now it is only used in `StandaloneEventRepository` (just for doSendMail) and hasn't been refactored,
leading to a poor pattern.

Therefore, I purposed to 
- rename `CalendarEventReplyPerformer` to `CalendarEventReplyMailProcessor` 
- create a new `CalendarEventAttendanceStatusPerformer` which acts as a bridge between JMAP methods and the service layer, making it reusable for accept, reject, and maybe method.